commands/omni_drive.py

```diff
@@ -0,0 +1,34 @@
+from wpilib.command import Command, CommandGroup
+
+class OmniDrive(Command):
+
+    def __init__(self, robot, name=None, timeout=None):
+        """This is the constructor of the command, use this to declare subsystem dependencies"""
+        #Use self.requires() here to declare subsystem dependencies
+        #eg. self.requires(chassis)
+        super().__init__(name, timeout)
+        self.requires(robot.chassis)
+        self.robot = robot
+
+    def initialize(self):
+        """Called just before this Command runs the first time"""
+        robot.drive(0.0, 0.0, 0.0, 0.0) # just to be safe
+
+    def execute(self):
+        """Called repeatedly when this Command is scheduled to run"""
+        # our axis are different from the wpilib, which is why vx vy and vz are getting different axis to
+        # the stick axis
+        #                      vX                           vY                           vZ                   throttle
```

Comment:
Whilst it is better to have detailed comments in the code, we should capture our approach externally such as a software plan (wiki or external word document).